src/app/search/search.component.html

```diff
@@ -39,6 +39,10 @@ <h1 id="pgTitle">Find Documents...</h1>
                     [disabled]="protoSearchActive"
                     (click)="dateRangeEnd.toggle();dateRangeStart.close();"
                     [(ngModel)]="terms.dateEnd"/>
+                    <span class="date-range__spacer"> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>
```

Comment:
I am guessing this will be subject to @jeznorth styling?

Reply:
Yes